core/pom.xml

```diff
@@ -66,6 +58,16 @@
             <groupId>org.scalaz</groupId>
             <artifactId>scalaz-core_${scala.compat.version}</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Do you need these at all? Aren't they inherited from spline-spark-api?

Reply:
yes for the sample project at least

Reply:
What I mean is that ```spline-core``` inherits ```spark-core_${scala.compat.version}``` and ```spark-sql_${scala.compat.version}``` from ```spline-core-spark-adapter-api```. So you shouldn't need to specify them again in ```spline-core```.

Reply:
sry actually provided scope is not transitive